app/scripts/components/common/chart/analysis/utils.ts

```diff
@@ -226,3 +227,15 @@ export function exportCsv(
     `${filename}.csv`
   );
 }
+
+const dateFormatsPerDensity = {
+  day: '%Y/%m/%d',
+  month: '%Y/%m',
+  year: '%Y',
+  default: '%Y/%m'
+};
+
+export function getTimeDensityFormat(td: TimeDensity | undefined) {
```

Comment:
(nitpick) 
```suggestion
export function getTimeDensityFormat(td?: TimeDensity) {
```

Reply:
ohh did not know about this trick. thanks!